scripts/world_bank/wdi/worldbank.py

```diff
@@ -95,7 +241,12 @@ def read_worldbank(iso3166alpha3, fetchFromSource):
                        delay=20,
                        backoff=1.5)
         if r.status_code != 200:
-            logging.info('Failed to retrieve %s', iso3166alpha3)
+            logging.fatal('Failed to retrieve %s', iso3166alpha3)
+        if not os.path.exists(os.path.join(_MODULE_DIR, 'raw_data')):
```

Comment:
can we call this `source_data` instead of `raw_data` ?

Reply:
Updated to source_data